### ISSUE TITLE
Problem with extracting onFocus event

### ISSUE THREAD

**Issue by svetlanapuro** [nobbynobbs/react-dadata-suggestions#28]
Hello,

I'm using react-dadata-suggestions with react-final-form (RFF) as a custom component. RFF has some [FieldRenderProps](https://final-form.org/docs/react-final-form/types/FieldRenderProps), which can be used with inputs. I'm using `input.value` and `meta.active` to define label classes like this:

```
  const labelClasses = classnames(
    "input-label",
    {
      "input-label--active": input.value !== "" || meta.active
    },
    className
  );
```

In case with DadataSuggestions `input.value` works, but `meta.active` (which shows whether or not the field currently has focus) doesn't.

Here is a [codesandbox](https://codesandbox.io/s/react-dadata-suggestions-components-bqz37?fontsize=14). Currently address input doesn't get `input-label--active` class onFocus, but only when there is some value.

(There is also an example of another custom component, where both `input.value` and `meta.active` work.)

Is there anyway to fix it?

**Comment by nobbynobbs** [comment]
I don't know. But I can add onFocus event into dadata component public api if RFF uses it under the hood

**Comment by svetlanapuro** [comment]
Thank you, maybe it will help.

I guess, here is how it's done in another library: https://github.com/sanniassin/react-input-mask/blob/1af73d764d59b39514dc40f0ec697ce76c3f335e/src/index.js#L451.

**Comment by nobbynobbs** [comment]
Fixed in v1.2.0.

**Comment by svetlanapuro** [comment]
Thank you very much 😊
